tests/test_core.py

```diff
@@ -102,17 +103,8 @@ def test_connect_args():
     host:port
     """
     engine = create_engine(
-        "snowflake://{user}:{password}@{host}:{port}/{database}/{schema}"
-        "?account={account}&protocol={protocol}".format(
-            user=CONNECTION_PARAMETERS["user"],
-            account=CONNECTION_PARAMETERS["account"],
-            password=CONNECTION_PARAMETERS["password"],
-            host=CONNECTION_PARAMETERS["host"],
-            port=CONNECTION_PARAMETERS["port"],
-            database=CONNECTION_PARAMETERS["database"],
-            schema=CONNECTION_PARAMETERS["schema"],
-            protocol=CONNECTION_PARAMETERS["protocol"],
-        )
+        f"""snowflake://{CONNECTION_PARAMETERS["user"]}:{CONNECTION_PARAMETERS["password"]}@{CONNECTION_PARAMETERS["host"]}:{CONNECTION_PARAMETERS["port"]}/{CONNECTION_PARAMETERS["database"]}/{CONNECTION_PARAMETERS["schema"]}"
+        "?account={CONNECTION_PARAMETERS["account"]}&protocol={CONNECTION_PARAMETERS["protocol"]}"""
```

Comment:
I believe that this was a mistake
```suggestion
        f"""snowflake://{CONNECTION_PARAMETERS["user"]}:{CONNECTION_PARAMETERS["password"]}@{CONNECTION_PARAMETERS["host"]}:{CONNECTION_PARAMETERS["port"]}/{CONNECTION_PARAMETERS["database"]}/{CONNECTION_PARAMETERS["schema"]}\
        ?account={CONNECTION_PARAMETERS["account"]}&protocol={CONNECTION_PARAMETERS["protocol"]}"""
```